### PR TITLE
Swagger apiview

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml [hunk 1]
```diff
@@ -375,3 +375,4 @@
     </div>
 </div>
 
+
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml:378]: nit: Remove this empty new line

File: src/dotnet/APIView/APIViewWeb/Startup.cs [hunk 2]
```diff
@@ -94,6 +94,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<LanguageService, GoLanguageService>();
             services.AddSingleton<LanguageService, ProtocolLanguageService>();
             services.AddSingleton<LanguageService, SwiftLanguageService>();
+            services.AddSingleton<LanguageService, SwaggerLanguageService>();
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Startup.cs:97]: This is not required in this PR.

**Comment by ruowan** [comment]
The swagger language service file support build navigationIds. If I don't use swagger language service. The functionality is broken.

File: tools/apiview/parsers/swagger-api-parser/swagger-api-parser/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+*.swagger
```
[the remark this answers is not in the document]
Comment (ruowan) [reply tools/apiview/parsers/swagger-api-parser/swagger-api-parser/.gitignore:1]: I want to ignore my local test fixtures.

**Review by praveenkuttappan — APPROVED** [review]

**Comment by praveenkuttappan** [comment]
I see that we don't have a pipeline to create and test this package. Let's get this merged and we can create a pipeline to test and publish this package.

**Pull request merged**